tools/build.py

```diff
@@ -425,7 +437,7 @@ def run_checktest(options):
     build_args = ['quiet=' + checktest_quiet]
     if options.iotjs_exclude_module:
         skip_module = ','.join(options.iotjs_exclude_module)
-        build_args.append('skip-module=' + skip_module)
+        skip_module and build_args.append('skip-module=' + skip_module)
```

Comment:
Your reading is reasonable, but the line was called. And it was because `options.iotjs_exclude_module` was not filtered when its list is empty. I fixed it. Thank you.